.travis.mk

```diff
@@ -317,6 +317,22 @@ test_oos_build:
 		-i ${DOCKER_IMAGE_TARANTOOL} \
 		make -f .travis.mk ${OOS_BUILD_RULE}
 
+# Odroid arm64
```

Comment:
Minor: I would also add `arm64` suffix for all targets below (as you had done for M1 targets).

Reply:
Done.